Description: Change session serializer to be compatible with Django 1.6
Author: Mathieu Comandon <strycore@gmail.com>
Index: django-openid-auth-0.5/example_consumer/settings.py
===================================================================
--- django-openid-auth-0.5.orig/example_consumer/settings.py	2014-01-14 13:13:58.223539726 +0900
+++ django-openid-auth-0.5/example_consumer/settings.py	2014-01-14 13:28:17.901040761 +0900
@@ -143,3 +143,7 @@
 
 # Should django_auth_openid be used to sign into the admin interface?
 OPENID_USE_AS_ADMIN_LOGIN = False
+
+# Modify temporarily the session serializer because the json serializer in
+# Django 1.6 can't serialize openid.yadis.manager.YadisServiceManager objects
+SESSION_SERIALIZER = 'django.contrib.sessions.serializers.PickleSerializer'
